Explorer/Assets/Scripts/SceneRuntime/Apis/Modules/RestrictedActionsApi/RestrictedActionsAPIWrapper.cs

```diff
@@ -53,8 +56,17 @@ public void Dispose()
             api.TryTriggerEmote(predefinedEmote);
 
         [UsedImplicitly]
-        public bool TriggerSceneEmote(string src, bool loop) =>
-            api.TryTriggerSceneEmote(src, loop);
+        public object TriggerSceneEmote(string src, bool loop)
+        {
+            async UniTask<bool> TriggerSceneEmoteAsync(CancellationToken ct)
+            {
+                try { return await api.TryTriggerSceneEmoteAsync(src, loop, ct); }
+                catch (Exception) { return false; }
```

Comment:
I believe you should an exception here if it occurs and if it is not Cancellation, otherwise it's completely ignored